glslc/src/file_compiler.h

```diff
@@ -101,6 +105,23 @@ class FileCompiler {
   };
 
  private:
+  enum class OutputType {
+    SpirvBinary,  // A binary module, as defined by the SPIR-V specification.
+    SpirvAssemblyText,  // Assembly syntax defined by the SPIRV-Tools project.
+    PreprocessedText,   // Preprocessed source code.
+  };
+
+  // Compiles the given string a the specified shader kind and using stored
```

Comment:
This blurb seems incorrect: `EmitCompiledResult()` doesn't compile anything, it merely emits the result of a prior compilation.